spec/bolt/node/winrm_spec.rb

```diff
@@ -10,7 +10,7 @@
   include BoltSpec::Files
 
   let(:host) { ENV['BOLT_WINRM_HOST'] || 'localhost' }
-  let(:port) { ENV['BOLT_WINRM_PORT'] || 55985 }
+  let(:port) { ENV['BOLT_WINRM_PORT'] || 5985 }
```

Comment:
I think this was 55985 because the port is forwarded in the Vagrantfile.

Reply:
Oops, removed that change.